ds701_book/_quarto-slides.yml

```diff
@@ -0,0 +1,37 @@
+project:
+  output-dir: _revealjs
+  pre-render: print_quarto_config.py
+  render:
+    - 01-Intro-to-Python.qmd
+    - 02A-Git-Jupyter.qmd
+    - 02B-Pandas.qmd
+    - 03-Probability-and-Statistics-Refresher.qmd
+    - 04-Linear-Algebra-Refresher.qmd
+    - 05-Distances-Timeseries.qmd
+    - 05A-Learning.qmd
+    - 06-Clustering-I-kmeans.qmd
+    - 07-Clustering-II-in-practice.qmd
+    - 08-Clustering-III-hierarchical.qmd
+    - 09-Clustering-IV-GMM-EM.qmd
+    - 10-Low-Rank-and-SVD.qmd
+    - 11-Dimensionality-Reduction-SVD-II.qmd
+    - 12-Anomaly-Detection-SVD-III.qmd
+    - 13-Learning-From-Data.qmd
+    - 14-Classification-I-Decision-Trees.qmd
+    - 15-Classification-II-kNN.qmd
+    - 16-Classification-III-NB-SVM.qmd
+    - 17-Regression-I-Linear.qmd
+    - 18-Regression-II-Logistic.qmd
+    - 19-Regression-III-More-Linear.qmd
+    - 20-Recommender-Systems.qmd
+    - 21-Networks-I.qmd
+    - 22-Networks-II-Centrality-Clustering.qmd
+    #- 23-NN-I-Gradient-Descent.qmd
+    - 24-NN-II-Backprop.qmd
+    - 25-NN-III-CNNs.qmd
+    - "!23-NN-I-Gradient-Descent.qmd" # for now ignore this file since cell 10 is stuck
```

Comment:
If it is complaining about errors in a code block, you can always have the site render and the error displayed in the writing by including in the code block

#| error: true

see https://quarto.org/docs/computations/execution-options.html

Reply:
Thanks. What was happening was the code block was taking a very long time to complete. I'll go back later and check why.